docs/references/admin.rst

```diff
@@ -83,12 +84,32 @@ Restart the database and watch the log file in real-time to understand how HTTP
 Server Version
 --------------
 
-When debugging a problem it's important to verify the PostgREST version. Look for the :code:`Server` HTTP response header, which contains the version number.
+When debugging a problem it's important to verify the running PostgREST version. There are 3 ways you can check the version:
+
+- Look for the :code:`Server` HTTP response header, which contains the version number. This header will be sent on all the requests.
```

Comment:
```suggestion
- Look for the :code:`Server` HTTP response header that PostgREST returns on every request.
```

I think that "which contains the version number" is implied in the previous paragraph.

Reply:
```suggestion
- Look for the :code:`Server` HTTP response header that is returned on every request.
```